model/types.go

```diff
@@ -284,7 +284,7 @@ type AccessInfo struct {
 type BlockDeviceAccessInfo struct {
 	AccessProtocol string   `json:"access_protocol,omitempty"`
 	TargetNames    []string `json:"target_names,omitempty"`
-	LunID          int32    `json:"lun_id,omitempty"`
+	LunID          string   `json:"lun_id,omitempty"`
```

Comment:
This change needs to make its way into the CSP spec definition here: https://github.com/hpe-storage/container-storage-provider/blob/master/spec.md.

Reply:
Raised PR -- https://github.com/hpe-storage/container-storage-provider/pull/14 for this spec change.